core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2596,7 +2596,7 @@ private[spark] object Utils extends Logging {
    * Redact the sensitive values in the given map. If a map key matches the redaction pattern then
    * its value is replaced with a dummy text.
    */
-  def redact(regex: Option[Regex], kvs: Seq[(String, String)]): Seq[(String, String)] = {
+  def redact[K, V](regex: Option[Regex], kvs: Seq[(K, V)]): Seq[(K, V)] = {
```

Comment:
@jzhuge . Please update the PR title with this main change.

Reply:
done